libs/openchallenges/home/src/lib/challenge-search/challenge-search.component.html

```diff
@@ -74,7 +74,7 @@ <h1>
     <div class="metrics-text-box">
       <h2
         *ngIf="orgCount$ | async as orgCount"
-        [countUp]="12"
+        [countUp]="13"
```

Comment:
This value should be fetched from the backend.

Reply:
@rrchai can you help with this? ^

Reply:
sure, I have created a ticket (#2241) regarding this task in my TO-DO list later this week~

Reply:
Thank you!! I appreciate it 🙏🏼